slave/process-mailman-cesnet-user-mail/changelog

```diff
@@ -0,0 +1,6 @@
+perun-slave-process-mailman-cesnet-user-mail (3.0.0) stable; urgency=low
+
+  * Copied implementation of perun-slave-process-mailman-cesnet
```

Comment:
Probably would be great to have also default purpose of this service even if it is just a copy of the existing service.

Reply:
More info added.